activejob/test/jobs/multiple_kwargs_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require_relative "../support/job_buffer"
+
+class MultipleKwargsJob < ActiveJob::Base
```

Comment:
I can modify https://github.com/rails/rails/blob/90982d850eea56a97ff7c3b6dacba61d60729741/activejob/test/jobs/kwargs_job.rb to accept two keyword arguments and not add this new class instead. I'd have to modify one existing test if I do that and I wasn't sure if we do that so I added a new class instead.